src/prommis/uky/uky_flowsheet.py

```diff
@@ -1020,103 +1607,431 @@ def initialize_system(m):
+                # Unfix feed states
+                m.fs.solex_cleaner_strip.mscontactor.organic_inlet_state[
+                    0
+                ].flow_vol.unfix()
+                m.fs.solex_cleaner_strip.mscontactor.aqueous_inlet_state[
+                    0
+                ].flow_vol.unfix()
+                m.fs.solex_cleaner_strip.mscontactor.organic_inlet_state[
+                    0
+                ].conc_mass_comp.unfix()
+                m.fs.solex_cleaner_strip.mscontactor.aqueous_inlet_state[
+                    0
+                ].conc_mass_comp.unfix()
         elif stream == m.fs.precipitator:
             print(f"Initializing {stream}")
             try:
-                initializer2.initialize(m.fs.precipitator)
+                initializer_bt.initialize(m.fs.precipitator)
             except:
                 # Fix feed states
                 m.fs.precipitator.cv_aqueous.properties_in[0].flow_vol.fix()
                 m.fs.precipitator.cv_aqueous.properties_in[0].conc_mass_comp.fix()
+                m.fs.precipitator.cv_precipitate.properties_in[0].flow_mol_comp.fix()
                 # Re-solve precipitator unit
                 solver = SolverFactory("ipopt")
                 solver.solve(m.fs.precipitator, tee=True)
                 # Unfix feed states
                 m.fs.precipitator.cv_aqueous.properties_in[0].flow_vol.unfix()
                 m.fs.precipitator.cv_aqueous.properties_in[0].conc_mass_comp.unfix()
+                m.fs.precipitator.cv_precipitate.properties_in[0].flow_mol_comp.unfix()
+        elif stream == m.fs.sl_sep2:
+            print(f"Initializing {stream}")
+            try:
+                initializer_bt.initialize(m.fs.sl_sep2)
+            except:
+                # Fix feed states
+                m.fs.sl_sep2.liquid_inlet_state[0].flow_vol.fix()
+                m.fs.sl_sep2.liquid_inlet_state[0].conc_mass_comp.fix()
+                m.fs.sl_sep2.solid_state[0].flow_mol_comp.fix()
+                # Re-solve unit
+                solver = SolverFactory("ipopt")
+                solver.solve(m.fs.sl_sep2, tee=True)
+                # Unfix feed states
+                m.fs.sl_sep2.liquid_inlet_state[0].flow_vol.unfix()
+                m.fs.sl_sep2.liquid_inlet_state[0].conc_mass_comp.unfix()
+                m.fs.sl_sep2.solid_state[0].flow_mol_comp.unfix()
+        elif stream == m.fs.precip_sep:
+            print(f"Initializing {stream}")
+            initializer_sep.initialize(m.fs.precip_sep)
+        elif stream == m.fs.precip_sx_mixer:
+            print(f"Initializing {stream}")
+            initializer_mix.initialize(m.fs.precip_sx_mixer)
         else:
             print(f"Initializing {stream}")
-            initializer2.initialize(stream)
+            initializer_bt.initialize(stream)
 
     seq.run(m, function)
 
 
-def solve(m):
+def solve(m, solver=None):
     """
-    Solve the system with IPOPT.
+    Solve the scaled system.
```

Comment:
It is not really accurate to say "scaled" model here - it will solve whatever model is passed in as `m`.